jbehave-core/src/test/java/org/jbehave/core/parsers/RegexCompositeParserBehaviour.java

```diff
@@ -66,6 +67,20 @@ public void shouldParseTwoCompositeStep() {
                 Collections.singletonList("Then another step"));
     }
 
+    @Test
+    public void shouldParseTwoCompositeStepWithCRLF() {
+        String compositeStepsAsText = "Composite: Given the first composite step" + CRLF+
+                "Given a step" + CRLF +
```

Comment:
Just in case, make it mixed: use  \n  and \r\n

Reply:
already present above in the other test